apps/console/src/features/connections/components/edit/forms/helpers/form-fields-helper.tsx

```diff
@@ -493,9 +498,74 @@ export const getDropDownField = (eachProp: CommonPluggableComponentPropertyInter
         </>
     );
 };
+/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
Let's add a new line before the comment.

btw, curious to know the reason for disabling eslint hook rules here :)

```suggestion

/* eslint-disable react-hooks/rules-of-hooks */
```